invenio_communities/communities/resolver.py

```diff
@@ -46,6 +47,7 @@ def pick_fields(community_dict):
             "description": metadata.get("description"),
             "type": metadata.get("type"),
         },
+        "visibility": access["visibility"]
```

Comment:
try to keep the same structure when expanding the fields i.e expand should look like
```
{...., "access": {"visibility": access.get("visibility")}
```